src/locale/index.js

```diff
@@ -24,6 +24,11 @@ const UA = {
       value: 'Polski',
     },
   ],
+  addItems: {
+    book: 'книжку',
```

Comment:
let's add with title case and use getLocale case option if have to add another case